cheroot/connections.py

```diff
@@ -49,6 +50,71 @@ def prevent_socket_inheritance(sock):
         fcntl.fcntl(fd, fcntl.F_SETFD, old_flags | fcntl.FD_CLOEXEC)
 
 
+class _ThreadsafeSelector:
+    """Threadsafe wrapper around a DefaultSelector.
```

Comment:
Looks like the spell-checker doesn't like a few things: https://github.com/cherrypy/cheroot/pull/337/checks?check_run_id=1440728038#step:9:277

```suggestion
    """Thread-safe wrapper around a DefaultSelector.
```

Reply:
This change was accepted inline. In the future, if you're accepting someone's change, consider also resolving the comment.

Reply:
@jaraco I usually recommend GH UI because it preserves the authorship and makes it easier to track what's accepted. Should we mention this in the contributing doc?

Reply:
I also recommend the GH UI, although that approach runs contrary to the other recommendation to 'squash related commits'. I personally have no objection to contributors amending commits with suggested changes, especially nitpicks or other minor edits. I mainly wished here to publish my surprise when I found that the comments were unresolved, but the changes were accepted. I'm okay without any guidance (just gentle nudges) or if you feel fervent about it, contributor guidance might be helpful. I do try to keep it short and sweet, as it's important to me to keep the barrier to contribution low.

Reply:
Happy to abide by whatever you prefer for this project - I found this unexpected in this case, though, since historically github has auto-collapsed (if not actually resolved) comments on code that has been modified in subsequent commits. Now they appear to only be labeled `Outdated`. Perhaps that has been changed in favor of explicit resolution of comments?

Reply:
@liamstask they appear outdated if changed in the branch. But they'd appear resolved if the UI button was used. Also, if you accept changes from the UI, it preserves the authorship of the person who submitted the suggested change. I usually accept suggestions via UI, then do pull to my local repo, and then squash or rearrange things as necessary.